utils/appliance.py

```diff
@@ -1117,6 +1124,11 @@ def has_netapp(self):
         return self.ssh_client().appliance_has_netapp()
 
     @lazycache
+    def guid(self):
```

Comment:
I'm not entirely sure that GUID always exists on appliances, but if it does, this could probably help us out in a few places where we're having trouble selecting exactly the right appliance to handle a task.